src/helpers/lockdrop/BitcoinLockdrop.ts

```diff
@@ -162,26 +162,64 @@ export function getPublicKey(address: string, signature: string, compression?: '
     return compression === 'compressed' ? compressedPubKey : uncompressedPubKey(compressedPubKey);
 }
 
+/**
+ * Validates the given BTC address by checking if it's in the correct format
+ * @param address Bitcoin public address
+ */
+function validateBtcAddress(address: string) {
+    try {
+        bitcoinjs.address.toOutputScript(address);
+        return true;
+    } catch (e) {
+        return false;
+    }
+}
+
 /**
  * used for CHECKSEQUENCEVERIFY relative time lock.
  * this converts days to bip68 encoded block number.
  * @param days number of days to be converted to sequence number
  */
 export function daysToBlocks(days: number) {
```

Comment:
What's about to check maximum amount of days here?

Reply:
@akru sorry for the late response. I've added the requested feature in this commit, please check https://github.com/staketechnologies/lockdrop-ui/pull/60/commits/8b85e5d5b03c737cf8da470854cd247cfb7e5a70